docs/examples/random.md

```diff
@@ -0,0 +1,75 @@
+# opsdroid skill-random
+
+A skill for [opsdroid](https://github.com/opsdroid/opsdroid) to perform random events like flipping a coin or rolling a dice.
+## Creating skills
+
+Skills are designed to be simpler than other modules to ensure that it is easy to get started.
+
+To create a skill you need to at minimum create a single python file in your repository with the  `__init__.py`  name. For example the skill  `hello`  has a single file called  `__init__.py`.
+
+##  Configuration
+```yaml
+None.
+```
```

Comment:
This section should not say `None.`.

How does the user configure opsdroid to use their random chance skill? Tell them that here.